### ISSUE TITLE
Update to Spring Boot 2 / Spring Cloud Greenwich

### ISSUE THREAD

**Issue by gdubya** [donovanmuller/spring-cloud-deployer-openshift#57]
This task is for upgrading the deployer to Spring Cloud Greenwich to provide support for Spring Boot 2.1.x and Java 11 compatibility.

I have started work on a fork, but I'm not sure how much work is involved.

**Comment by donovanmuller** [comment]
@gdubya I'll take a look. Thanks 👍

When you have completed this task, would you consider opening a PR?
I no longer have enough time to maintain this project unfortunately, so any help is much appreciated.

**Comment by gdubya** [comment]
Thanks. The unit tests appear to be ok now, but not sure how to get the integration tests. Is this possible with minishift?

Yes, i'll submit a PR when it's ready. :+1:

**Comment by donovanmuller** [comment]
Yes, I normally run the integration tests against a local minishift instance. Can take a while so be patient...
